README.md

```diff
@@ -53,18 +53,39 @@ Now that we've covered the basics of using `dotnew new`, lets move on to info fo
 If you're authoring templates, or interested in contributing to this repo, then you're likely interested in how to use the latest version of this experience.
 The steps required are outlined below.
 
-Step 1: Get the SDK for your platform from [dotnet/cli](https://github.com/dotnet/cli)
+## Prerequisites
+In order to work with this project, you will need the following.
+- [Visual Studio 2017](https://www.visualstudio.com/)
 
-Step 2: Clone this repository to your local machine
+## Setup
 
-Step 3:  Run the setup script
- - **Windows:** [setup.cmd](https://github.com/dotnet/templating/blob/master/setup.cmd)
- - **Mac/Linux**: [setup.sh](https://github.com/dotnet/templating/blob/master/setup.sh) 
+Step 1:
+- Get the SDK for your platform from [dotnet/cli](https://github.com/dotnet/cli).
+- Typically, development is  done against the latest RC FTS Release.  
+  - It is best to ask the team to clear up any compatibility concerns before starting.
```

Comment:
The SDK is tricky in this regard and has a different role to play in different development scenarios.

1) Running setup at the command prompt - the appropriate SDK is downloaded into the `artifacts/.dotnet` directory and is used to do the build. This means that the globally installed SDK (if any) doesn't have influence on the build itself
2) Calling `dotnet new3` - the version of the SDK doesn't matter here, the command resolution behavior that is able to locate `dotnet-new3.exe` (or appropriate variants for other platforms) has been built in to the dotnet CLI for a very long time
3) Working with the source outside of Visual Studio - no particular SDK is required here as `setup` takes care of getting the right SDK for you
4) Working with the source in Visual Studio - this is the place where the version of the SDK matters, but the behavior isn't influenced by the version of the dotnet CLI. A portion of the SDK also ships in Visual Studio (tasks and targets) which governs what files display, when they display, the type of project that each csproj should be interpreted as being, etc. The appropriate guidance here is to use the latest available version of Visual Studio 2017.